<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="GameMenu">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="firstPlayerSet">
     <item>
      <widget class="QLabel" name="firstLabel">
       <property name="text">
        <string>First Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fHuman">
       <property name="text">
        <string>Human</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fMonte">
       <property name="text">
        <string>AI (Monte Carlo)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fRand">
       <property name="text">
        <string>AI (Random)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="secondPlayerSet">
     <item>
      <widget class="QLabel" name="secondLabel">
       <property name="text">
        <string>Second Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sHuman">
       <property name="text">
        <string>Human</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sMonte">
       <property name="text">
        <string>AI (Monte Carlo)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sRand">
       <property name="text">
        <string>AI (Random)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sizeSet">
     <item>
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sizeButton_7">
       <property name="text">
        <string>7</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sizeButton_9">
       <property name="text">
        <string>9</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sizeButton_11">
       <property name="text">
        <string>11</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHEX">
    <property name="title">
     <string>HEX</string>
    </property>
   </widget>
   <addaction name="menuHEX"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
